tests/release/e2e-test-default-bundle.go

```diff
@@ -0,0 +1,249 @@
+			Expect(err).NotTo(HaveOccurred())
+		})
+
+		It("CreateComponent", func() {
+			_, err := framework.HasController.CreateComponent(applicationNameDefault, componentName, devNamespace, gitSourceComponentUrl, "", containerImageUrl, "", "", false)
+			Expect(err).NotTo(HaveOccurred())
+		})
+	})
+
+	var _ = Describe("Post-release verification", func() {
+
+		It("verifies that in dev namespace will be created a PipelineRun.", func() {
+			Eventually(func() bool {
+				prList, err := framework.TektonController.ListAllPipelineRuns(devNamespace)
+				if err != nil || prList == nil || len(prList.Items) < 1 {
+					klog.Error(err)
+					return false
+				}
+				return strings.Contains(prList.Items[0].Name, componentName)
+			}, releasePipelineRunCreationTimeout, defaultInterval).Should(BeTrue())
+		})
+
+		It("verifies that the PipelineRun in dev namespace succeeded.", func() {
+			Eventually(func() bool {
+				prList, err := framework.TektonController.ListAllPipelineRuns(devNamespace)
+				if prList == nil || err != nil || len(prList.Items) < 1 {
+					klog.Error(err)
+					return false
+				}
+
+				return prList.Items[0].HasStarted() && prList.Items[0].IsDone() && prList.Items[0].Status.GetCondition(apis.ConditionSucceeded).IsTrue()
+			}, releasePipelineRunCompletionTimeout, defaultInterval).Should(BeTrue())
+		})
+
+		// TODO should be removed once the it's done by service
+		// Adds manually the label of PaC to created snapshot in dev namespace
+		It("gets snapshot created in dev namepsace and add labebl to the Snapshot.", func() {
+			snapshotCreatedInDev, err := framework.ReleaseController.GetSnapshotInNamespace(devNamespace, componentName)
+			Expect(err).NotTo(HaveOccurred())
+			_, err = framework.ReleaseController.AddLabelToSnapshot(snapshotCreatedInDev)
+			Expect(err).NotTo(HaveOccurred())
+		})
+
+		It("verifies that in managed namespace will be created a PipelineRun.", func() {
+			Eventually(func() bool {
+				prList, err := framework.TektonController.ListAllPipelineRuns(managedNamespace)
+				if err != nil || prList == nil || len(prList.Items) < 1 {
+					klog.Error(err)
+					return false
+				}
+				return strings.Contains(prList.Items[0].Name, "release")
+			}, releasePipelineRunCreationTimeout, defaultInterval).Should(BeTrue())
+		})
+
+		It("verifies a PipelineRun started in managed namespace succeeded.", func() {
+			Eventually(func() bool {
+				prList, err := framework.TektonController.ListAllPipelineRuns(managedNamespace)
+				if prList == nil || err != nil || len(prList.Items) < 1 {
+					klog.Error(err)
+					return false
+				}
+				return prList.Items[0].HasStarted() && prList.Items[0].IsDone() && prList.Items[0].Status.GetCondition(apis.ConditionSucceeded).IsTrue()
+			}, releasePipelineRunCompletionTimeout, defaultInterval).Should(BeTrue())
+		})
+
+		It("tests a Release should have been created in the dev namespace and succeeded", func() {
```

Comment:
I'm not sure how this file is a test of the bundle

Reply:
To ensure that the release object is created and passed , 
there might be cases where due to some bug that PR pass and Release not ...